modules/vaos/config/routes.rb

```diff
@@ -20,7 +20,7 @@
       resources :limits, only: :index
       get 'visits/:schedule_type', to: 'visits#index'
     end
-    resources :preferences, only: :index
+    resource :preferences, only: %i[show update]
```

Comment:
This is confusing - maybe we need to consider a different name, if we're actually returning *one* resource that's a `PreferenceCollection` or something, otherwise the plurality doesn't make sense. Maybe I'm just not understanding the functionality on VAMF side correctly.

Reply:
I don't think it's confusing at all. I mean rails allows you to have inflections as you deem prudent for a reason. Sometimes things just read better one way or the other. In this case your resource is a single resource, that in spoken language is usually pluralized. It's not a collection though -- it's a set of attributes that characterize preferences. A collection would imply that every single attribute should have it's own ID. Thats not the case at all.

Reply:
`resource` vs `resources` distinguishes whether it is a singular resource or collection. How you choose to inflect is more a matter of language you wish to use.